api.ts

```diff
@@ -1144,6 +1145,23 @@ export interface SyncFormulaDef<
    * @hidden
    */
   updateOptions?: Pick<CommonPackFormulaDef<ParamDefsT>, 'extraOAuthScopes'>;
+
+  /**
+   * Given a set of rows, return permission lists, keyed by the rows' idProperty values.
+   *
+   * TODO(patrick): Unhide this
+   * @hidden
+   */
+  getPermissions?(rows: Array<ObjectSchemaDefinitionType<K, L, SchemaT>>): Promise<Record<string, Permission[]>>;
+
+  /**
+   * If the table implements {@link getPermission}, the maximum number of rows that will be sent to that
+   * function in a single batch. Defaults to 10 if not specified.
+   *
+   * TODO(patrick): Unhide this
+   * @hidden
+   */
+  maxPermissionBatchSize?: number;
```

Comment:
Would this be better as a parameter to getPermissions?

Reply:
Not sure how that would work... the pack maker defines the number, but coda code invokes getPermissions...